### PR TITLE
Corrections and additions to French i18n

### PR DESCRIPTION
**Pull request by boydkelly**
A few strings were untranslated.   One correction.  The expression 'Créer un problème' in French impies the user is the source of the problem.   The replacement verb 'signaler' has more the sens of 'Report an issue'.  (Maybe thats better in English too... but we won't go there!)

### CODE REVIEW

File: i18n/fr.toml [hunk 1]
```diff
@@ -38,18 +38,21 @@ other = "Modifier cette page"
 [post_create_child_page]
 other = "Create child page"
 [post_create_issue]
-other = "Créer un problème dans la documentation"
+other = "Signaler un problème dans la documentation"
 [post_create_project_issue]
-other = "Créer un problème dans le projet"
+other = "Signler un problème dans le projet"
```
Comment (LisaFC) [inline i18n/fr.toml:43]: Should be "Signaler"?

File: userguide/config.toml [hunk 2]
```diff
@@ -37,7 +37,7 @@ blog = "/:section/:year/:month/:day/:slug/"
 [outputs]
 home = [ "HTML" ]
 page = [ "HTML" ]
-section = [ "HTML", "RSS", "print"]
+#section = [ "HTML", "RSS", "print"]
```
Comment (LisaFC) [inline userguide/config.toml:40]: Did you intend to comment this out?

File: i18n/fr.toml (hunk 1 above)
Comment (boydkelly) [reply i18n/fr.toml:43]: aarrg!  typo!!!  Yes it should!

File: userguide/config.toml (hunk 2 above)
Comment (boydkelly) [reply userguide/config.toml:40]: No!   Sorry about that.   I thought I had squashed that from the commit.   (For some reason the print output was causing a hugo error.  Not sure what I did so just commented it for the time being.  Please ignore that.)

**Comment by LisaFC** [comment]
Looks good, thanks!

**Pull request merged**